.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,6 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
-    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```

Comment:
Please add this line back so that users use the guidelines in the repo and not those in their IDE.